packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -46,13 +46,23 @@ interface IMultiselectProps extends HTMLAttributes<HTMLDivElement> {
   validation?: VALIDATION;
   /** Determines the maximum number of items displayed while collapsed */
   maxItems?: number;
-  /** Callback that overrides the "+ N more" text displayed when the total number of items exceeds `maxItems` */
+  /**
+   * Overrides the "+ N more" text displayed when the total number of items exceeds `maxItems`
+   *
+   * @param {number} index
```

Comment:
Can this parameter get described? Something like "The number of hidden items". fwiw this parameter is not an `index`, it is more like a `count`. I think it would be pretty easy to change the signature below.

Reply:
Or, maybe, `value`? But not `index`.

Reply:
Ahh I see. Yes I was a little confused on this one. I can update the signature to value if that's alright with you